ros_buildfarm/status_page_input.py

```diff
@@ -98,5 +98,12 @@ def get_rosdistro_info(dist, build_file):
             except InvalidPackage:
                 pass
 
+        # handle ignored packages
+        if pkg_name not in filtered_pkg_names:
+            ros_pkg.version = None
+            ros_pkg.status = 'disabled'
+            ros_pkg.status_description = \
+                'disabled by the release build configuration'
```

Comment:
Is this text shown anywhere to the user?

Reply:
It is the alt-text when you hover over the 'disabled' text.

Reply:
I only see "<reponame>: None" in the tooltip?

Reply:
It's the alt text for the package status - the "disabled" text, not the build status on the right side.

Reply:
I'd guess the fact that the package is disabled by the build configuration is less of a reflection on the status of the package itself (such as maintained, unmaintained, etc).

I'm OK dropping the changes to `.status` and `.status_description` if you agree. The build status, which is changed to grey by setting `.version = None`, is the more important change from my perspective.

Reply:
Aha, no need to drop it. Now that I found it I think the information is helpful.